src/compute/src/sink/persist_sink.rs

```diff
@@ -85,10 +85,11 @@ where
         let persist_client = futures_executor::block_on(persist_location.open())
             .expect("cannot open persist client");
 
-        let (write, _read) = futures_executor::block_on(
+        let (write, read) = futures_executor::block_on(
```

Comment:
Should be changed to the new `open_writer`.

I'm pointing out obvious things here, just to leave a note. Hope that's ok ... 😅